ecommerce/OrderController.php

```diff
@@ -259,9 +259,8 @@ public function create_order( int $user_id, array $items, string $payment_status
 	 * @return void
 	 */
 	public function get_order_by_id() {
```

Comment:
This method is bind with ajax `add_action( 'wp_ajax_tutor_order_details', array( $this, 'get_order_by_id' ) );` and confusing with model method name. Rename it like `ajax_get_order_details` to enhance clarity of it's action.